ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blocks/blockbody/versions/capella/BeaconBlockBodyBuilderCapella.java

```diff
@@ -30,21 +28,20 @@
 
 public class BeaconBlockBodyBuilderCapella extends BeaconBlockBodyBuilderBellatrix {
 
-  private BeaconBlockBodySchemaCapellaImpl schema;
-  private BlindedBeaconBlockBodySchemaCapellaImpl blindedSchema;
+  private final BeaconBlockBodySchemaCapellaImpl schema;
+  private final BlindedBeaconBlockBodySchemaCapellaImpl blindedSchema;
   private SszList<SignedBlsToExecutionChange> blsToExecutionChanges;
 
-  public BeaconBlockBodyBuilderCapella schema(final BeaconBlockBodySchemaCapellaImpl schema) {
-    this.schema = schema;
-    this.blinded = Optional.of(false);
-    return this;
+  public BeaconBlockBodyBuilderCapella() {
+    this.schema = null;
+    this.blindedSchema = null;
   }
```

Comment:
if this does ever get called it's not going to end well...

Reply:
i guess to clarify, if we're going down this route, we really need to do some checks on build to make sure everything is as it needs to be - null of either of these will be brutal through most of the code if it gets to blockBody objects...

Reply:
I was thinking a lot how to deal with Builders default constructor. I'd suggest to mark it with `@Deprecated(since = "Not for use")` in each builder. I think it's the least evil to show a clear sign it should not be used.